app/models/miq_worker.rb

```diff
@@ -205,31 +206,39 @@ def self.fetch_worker_settings_from_options_hash(options_hash, raw = false)
       classes.each do |c|
         section = section[c]
         raise _("Missing config section %{section_name}") % {:section_name => c} if section.nil?
+
         defaults = section[:defaults]
         unless defaults.nil?
-          defaults.delete_if {|k, v| v == Vmdb::Settings::RESET_VALUE }
+          defaults.delete_if { |_k, v| v == Vmdb::Settings::RESET_VALUE }
           settings.merge!(defaults)
         end
       end
 
-      section.delete_if {|k, v| v == Vmdb::Settings::RESET_VALUE }
+      section.delete_if { |_k, v| v == Vmdb::Settings::RESET_VALUE }
       settings.merge!(section)
 
       # If not specified, provide the worker_settings cleaned up in fixnums, etc. instead of 1.seconds, 10.megabytes
       # Clean up the configuration values in a format like "30.seconds"
-      unless raw == true
-        settings.keys.each do |k|
-          if settings[k].kind_of?(String)
-            if settings[k].number_with_method?
-              settings[k] = settings[k].to_i_with_method
-            elsif settings[k] =~ /\A\d+(.\d+)?\z/ # case where int/float saved as string
-              settings[k] = settings[k].to_i
-            end
-          end
+      normalize_settings!(settings) unless raw == true
```

Comment:
/fwiw looks like `fetch_worker_settings_from_options_hash(..., raw: true)` can only ever be called by `fetch_worker_settings_from_server` - but `options[:raw]` is never passed.

So it looks like `raw` is always `true`.
which means...
absolutely nothing.

Reply:
A caller could pass it via `worker_settings(:raw => true)`, but you're right no one currently does as far as I can tell.

Reply:
The UI used to pass raw => true for advanced settings presentation (we don't want to auto-convert there)